core/arch/arm/mm/core_mmu.c

```diff
@@ -264,7 +264,7 @@ static struct tee_mmap_region *find_map_by_pa(unsigned long pa)
 	struct tee_mmap_region *map = get_memory_map();
 
 	while (!core_mmap_is_end_of_table(map)) {
-		if ((pa >= map->pa) && (pa < (map->pa + map->size)))
+		if ((pa >= map->pa) && (pa <= (map->pa + map->size - 1)))
```

Comment:
this change generates a checkpatch CHECK report:
```
8e98afaf3 (HEAD -> ECA-fix-32b) core: arm: mmu: fix find_map_by_pa() on areas end addresses
CHECK: Unnecessary parentheses around 'pa >= map->pa'
#23: FILE: core/arch/arm/mm/core_mmu.c:267:
+		if ((pa >= map->pa) && (pa <= (map->pa + map->size - 1)))

total: 0 errors, 0 warnings, 1 checks, 8 lines checked
```
Should I fix it?

Reply:
Either way works for me.

Reply:
ok, i'll keep as-is.